plugin/docker/src/main/java/com/perl5/lang/perl/idea/sdk/host/docker/PerlDockerAdapter.java

```diff
@@ -171,7 +196,7 @@ private void dropContainer(@NotNull String containerName) throws ExecutionExcept
 
   private @NotNull List<PerlFileDescriptor> doListFiles(@NotNull String containerName, @NotNull String path) throws ExecutionException {
     try {
-      ProcessOutput output = runCommand(EXEC, containerName, "ls", "-LAs", "--classify", path);
+      ProcessOutput output = runCommand(EXEC, INTERACTIVELY, containerName, "ls", "-LAs", "--classify", path);
```

Comment:
why `ls` command should be run interactively?

Reply:
I test it on my machine when i run without interactively i dont get any output in ls in command line so i get empty response. I use rancher-desktop on my machine and when i run on my machine under wsl i dont get any output.
Here is example what i get as output
```
igi@lenovo:~$ docker run -d --rm --entrypoint "" --name intellijPerl_copying perl bash -c "while true;do sleep 1000000;done"                                3ad9bd98fc4ed8004e68ded661b5c7d5aa99ecd44e2144fee568b879fc944a21                                                                                            igi@lenovo:~$ docker exec intellijPerl_copying ls -LAs --classify /                                                                                         igi@lenovo:~$ docker exec -i intellijPerl_copying ls -LAs --classify /                                                                                      total 72                                                                                                                                                     0 .dockerenv*                                                                                                                                              12 bin/                                                                                                                                                      4 boot/                                                                                                                                                     0 dev/                                                                                                                                                      4 etc/                                                                                                                                                      4 home/                                                                                                                                                     4 lib/                                                                                                                                                      4 lib64/                                                                                                                                                    4 media/                                                                                                                                                    4 mnt/                                                                                                                                                      4 opt/                                                                                                                                                      0 proc/                                                                                                                                                     4 root/                                                                                                                                                     4 run/                                                                                                                                                      4 sbin/                                                                                                                                                     4 srv/                                                                                                                                                      0 sys/                                                                                                                                                      4 tmp/                                                                                                                                                      4 usr/                                                                                                                                                      4 var/
```

when i run it in windows shell i dont need provide -i and i get output.

Reply:
I'm afraid that fixing this for your particular case is not enough. If you want to fix this yourself, you need to find the reason, describe it, fix the plugin and test.

Reply:
Probably i hit this with my setup: https://stackoverflow.com/questions/35550694/what-is-the-purpose-of-the-i-and-t-options-for-the-docker-exec-command/35551071#35551071

Reply:
And also  what is interesting if i provide -it in ls command i get our as one liner in terminal.

I installed docker-ce-cli and use it the same ;-/.

Reply:
Ok i see its bug in docker rancher....
https://github.com/rancher-sandbox/rancher-desktop/issues/3239 (so i will use till they fixed my local plugin build)

SO i close this pull request and create new one for copying resources when is relative symlink in perl @Inc directories just provide me link where test around docker integration is located so i can get inspiration.

Reply:
Ok, this seems a long standing issue and it is unclear when and if they will fix it. We could introduce the registry/settings option to control this may be.